runtime/src/lib.rs

```diff
@@ -358,17 +358,17 @@ impl session::historical::Trait for Runtime {
 srml_staking_reward_curve::build! {
     const REWARD_CURVE: PiecewiseLinear<'static> = curve!(
         min_inflation: 0_025_000,
-        max_inflation: 0_100_000,
-        ideal_stake: 0_500_000,
+        max_inflation: 0_300_000,
+        ideal_stake: 0_300_000,
         falloff: 0_050_000,
-        max_piece_count: 40,
+        max_piece_count: 100,
```

Comment:
@bwhm just wanted to bring your attention to this change of the max_piece_count. I increased it following change to max_inflation and ideal_stake because the unit test failure suggested so:
```
---- __srml_staking_reward_curve_test_module::reward_curve_precision stdout ----
thread '__srml_staking_reward_curve_test_module::reward_curve_precision' panicked at '
Generated reward curve approximation differ from real one:
	for i = 364050 and base = 1000000, f(i/base) * base = 138165,
	but approximation = 138856,
	err = 0005001 millionth,
	try increase the number of segment: 40 or the test_error: 5000.
```